geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ConnectCommand.java

```diff
@@ -238,11 +240,12 @@ private static boolean containsSSLConfig(Properties properties) {
   }
 
 
-  Result httpConnect(Properties gfProperties, String url) {
+  Result httpConnect(Properties gfProperties, String url, boolean skipSslVerification) {
     Gfsh gfsh = getGfsh();
     try {
       SSLConfig sslConfig = SSLConfigurationFactory.getSSLConfigForComponent(gfProperties,
           SecurableCommunicationChannel.WEB);
+      sslConfig.setSkipSslVerification(skipSslVerification);
```

Comment:
do not need to set the flag here if we pass the flag to the method call 2 lines below